src/widgets/WidgetMainHumm90.vue

```diff
@@ -0,0 +1,142 @@
+<template>
+  <div v-if="isWidgetOpen" class="widget">
+    <WidgetContent
+      @toggle-dialog="isDialogOpen = true"
+      @close-widget="isWidgetOpen = false"
```

Comment:
does it actually matter if you have `v-if="isWidgetOpen"` in a parent component? I don't think it will be rendered if `isWidgetOpen` is `false` anyway

Reply:
the v-if is required. I've moved the `.widget` div inside `WidgetContent`